README.md

```diff
@@ -1,88 +1,100 @@
-# lambda-template
-Template for creating lambda repositories
-A template for quickly starting a new AWS lambda project.
+# cfn-ssm-param-macro
 
-## Naming
-Naming conventions:
-* for a vanilla Lambda: `lambda-<context>`
-* for a Cloudformation Transform macro: `cfn-transform-<context>`
-* for a Cloudformation Custom Resource: `cfn-cr-<context>`
+The `SsmParam` macro allows retrieval of parameters from the SSM Parameter Store.
```

Comment:
A cloudformation macro is the same as a transform.  Lets follow the naming convention and rename this to `cfn-transform-ssm-param`

Reply:
i changed my mind, the documentation seems to refer to macros as an instance of transform.  I think it's more appropriate to change the name to `cfn-macro-sssm-param`

Reply:
`cfn-macro-sssm-param` or `cfn-macro-ssm-param` ? I don't see where the extra "s" would come from

Reply:
that's a typo @cascadianblue should be `cfn-macro-ssm-param`